### PR TITLE
Add a controller, route, and view for data

### PR DESCRIPTION
**Pull request by graysonwright**
This should allow the designers to start implementing in code,
while we set up authentication and data-fetching on the backend.

### CODE REVIEW

File: app/views/data/index.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>You're a speedy snowman!</h1>
```
Comment (graysonwright) [inline app/views/data/index.html.erb]: This page is where we'd be putting all of the graphs, driver archetype notes, etc.

File: app/views/pages/welcome.html.erb [hunk 2]
```diff
@@ -1 +0,0 @@
-<%= react_component("HelloWorld", name: "Grayson") %>
```
Comment (graysonwright) [inline app/views/pages/welcome.html.erb]: This was a leftover view from a proof-of-concept PR. We won't need it anymore.

File: app/controllers/data_controller.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class DataController < ActionController::Base
```
Comment (paulcsmith) [inline app/controllers/data_controller.rb]: I'm not sure about this name, but can't think of something better. Maybe `DataVisualizationsController`? If we can't think of anything I'm cool with merging this as-is

**Comment by paulcsmith** [comment]
Looks good!

**Comment by creuter** [comment]
:+1:

File: app/controllers/data_controller.rb (hunk 3 above)
Comment (graysonwright) [inline app/controllers/data_controller.rb]: @paulcsmith, take your pick:

- `DriverDataController`
- `DrivingDataController`
- `HolidayDataController`
- `VisualizationsController`
- `DrivingVisualizationsController`

Comment (paulcsmith) [inline app/controllers/data_controller.rb]: I'm a fan of DriverDataController personally

**Pull request merged**